rpc2/client.go

```diff
@@ -11,6 +11,7 @@ func NewClient(xp Transporter, f UnwrapErrorFunc) *Client {
 
 func (c *Client) Call(method string, arg interface{}, res interface{}) (err error) {
 	var d Dispatcher
+	c.xp.Run(true)
```

Comment:
`Transport.GetDispatcher` no longer calls `Transport.run`, so we need to call it explicitly.